indico/core/db/sqlalchemy/protection.py

```diff
@@ -354,6 +354,33 @@ def update_principal(self, principal, read_access=None, full_access=None, roles=
                                        old_data=old_data, quiet=quiet)
         return entry
 
+    def get_manager_list(self, recursive=False):
+        from MaKaC.conference import Category
+        managers = {x.principal for x in self.acl_entries if x.has_management_role()}
+        if recursive and self.protection_parent:
+            # XXX: Remove this condition check when moving Category to new models
+            if isinstance(self.protection_parent, Category):
+                managers.update(x.as_new for x in self.protection_parent.getRecursiveManagerList())
+            else:
+                managers.update(self.protection_parent.get_manager_list(recursive=True))
+        return managers
+
+    def get_access_list(self):
```

Comment:
doesn't this belong in the base `ProtectionMixin`? The only thing you lose is the ability to check for `if x.read_access` but since anyone in the ACL has read_access (the flag is just to make it more explicit) this would not be a problem